#Yesterdays assignment :Question1

Read a string from user and print last character of the string

In [1]:
str1 = input()
ch = str1[-1]
print(ch)

Gandhinagar


#Yesterdays assignment :Question 2
1. Read a string from user STR1.
2. 

(if the length of STR1 is less than or equal to 7)

    -> read 2 strings and print last character of the two strings

(if the length of string is greater than or equal to 7 and less than or equal to 10 )

      ->then read 3 float variables from user and print sum of that 4 variables

Otherwise
        -> print "String is too leangthy"

In [5]:

STR1 = 'Ahmedab'

l = len(STR1)
if(l<=7):
  s1 = input()
  s2 = input()
  print(s1[-1], s2[-1])
elif(l>=7 and l<=10):
  f1 = float(input())
  f2 = float(input())
  f3 = float(input())
  print(f1+f2+f3)
else:
  print("String is too leangthy")

abcd
pqr
d r


#Modulo operator

In [6]:
a = 10 
if(a%2 == 0):
  print("Even")
else:
  print("Odd")

2

#Nested if else 

Read integer from user 

if number is greater than 100

    if number is odd 

        then print (Big and ODD)
    if number is even

        then print (Big and Even)

if number is less than or equal to 100

      if number is odd 

        then print (Small and ODD)
        
     if number is even
        then print (Small and Even)


In [7]:
i = int(input())
if(i>100):
  if(i%2 == 1):
    print("Big and ODD")
  else:
    print("Big and Even")
else:
  if(i%2 ==1):
    print("Small and ODD")
  else:
    print("Small and Even")

23
Small and ODD


#For loop

In [10]:
#print numbers from 17 to 45
for i in range(17,46,1):
  print(i)

17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45


In [12]:
#Print even numbers from 1 to 50
for i in range(2,51,2):
  print(i)
  
#Alternative method
for i in range(1,51,1):
  if(i%2 == 0):
    print(i)

# Write table of 18 (from 1 to 10)

18 * 1 = 18

18 * 2 = 36

.
.

18 * 10 = 180

In [13]:
for i in range(1,11,1):
  print("18 *",i, "=", 18*i)

18 * 1 = 18
18 * 2 = 36
18 * 3 = 54
18 * 4 = 72
18 * 5 = 90
18 * 6 = 108
18 * 7 = 126
18 * 8 = 144
18 * 9 = 162
18 * 10 = 180


#Reverse for loop

In [15]:
for i in range(20,9,-1):
  print(i)

20
19
18
17
16
15
14
13
12
11
10


#Write a reverse for loop to print values from 100 to 10 who are divisible either 5 or 3

In [16]:
for i in range(100,9,-1):
  if(i%5==0 or i%3 ==0):
    print(i)

100
99
96
95
93
90
87
85
84
81
80
78
75
72
70
69
66
65
63
60
57
55
54
51
50
48
45
42
40
39
36
35
33
30
27
25
24
21
20
18
15
12
10


#Assignment 1:
#Write a table of 18 (from 1 to 30) for numbers divisible by 3 only


18 * 3 = 54

18 * 6 = 108

..

18 * 30 = 540


#Assignment 2
#Read integer and print digit of the integer (Its not done by for loop)

Example1 : 8 : Answer -1

Example2 : 204 : Answer - 3

Example2 : 20455 : Answer - 5


#Assignment 3

Write numbers from 500 to 100 who are divisible by 5 but not divisible by 25